[AAP-38821] Namespace-Scoped Deployments in OCP and Topology (#2910)
ansible/aap-docs: downstream/assemblies/platform/assembly-install-aap-operator.adoc

= Installing the {OperatorPlatformName} on {OCP}

[role="_abstract"]
When installing your {OperatorPlatformNameShort} you have a choice of a namespace-scoped operator or a cluster-scoped operator.

> This depends on the update channel you choose, stable-2.x or cluster-scoped-2.x.

Reviewer: Does the choice depend on the channel?

Author: Yes, if the select stable-2.x for the change it installs a namespace-scoped operator, and if they select cluster-scoped-2.x they will get a cluster scoped operator.

> A namespace-scoped operator is confined to one namespace, offering tighter security. While a cluster-scoped operator spans multiple namespaces, which grants broader permissions.

Reviewer: ```suggestion
A namespace-scoped operator is confined to one namespace, offering tighter security. A cluster-scoped operator spans multiple namespaces, which grants broader permissions.
```

If you are managing multiple {PlatformNameShort} instances with the same {OperatorPlatformNameShort} version, use the cluster-scoped operator, which uses a single operator to manage all {PlatformNameShort} custom resources in your cluster. 

> If you need multiple operator versions in the same cluster, it is necessary to use the namespace-scoped operator.

Reviewer: ```suggestion
If you need multiple operator versions in the same cluster, you must use the namespace-scoped operator.
```

> The operator and the deployment will share the same namespace.

Reviewer: ```suggestion
The operator and the deployment share the same namespace.
```

> This can also be helpful when debugging as the operator logs will only pertain to custom resources in that namespace.

Reviewer: ```suggestion
This can be helpful when debugging because the operator logs pertain to custom resources in that namespace only.
```
Someone once told me in a review that the best place to put "only" is next to the unique thing it describes.
So the logs aren't only for custom resources (?), but the only custom resources in the logs are the CRs that are in that namespace.
